go.mod

```diff
@@ -60,6 +60,7 @@ require (
 	github.com/ipfs/interface-go-ipfs-core v0.5.2
 	github.com/ipfs/tar-utils v0.0.2
 	github.com/ipld/go-car v0.3.2
+	github.com/ipld/go-car/v2 v2.1.1 // indirect
```

Comment:
Why is this an indirect import, it seems like we're directly using it? Try running `go mod tidy`.

Reply:
Good catch! Should be fixed now.